packages/react-components/react-migration-v8-v9/stories/CheckboxShim/Description.md

```diff
@@ -0,0 +1 @@
+The CheckboxShim component accepts the same props as a v8 Checkbox and renders a v9 Checkbox.
```

Comment:
can we provide more comprehensive docs on par with other shim components ? see for exmaples https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-components-textarea-migration--page

Reply:
> can we provide more comprehensive docs on par with other shim components ? see for exmaples https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-components-textarea-migration--page

Yes, already added `CheckboxShim` example and props mappings in https://github.com/microsoft/fluentui/blob/449d2f0712056ac0457169af053b71ee0aff95a7/apps/public-docsite-v9/src/Concepts/Migration/FromV8/Components/Checkbox.stories.mdx
Really thanks for your suggestion